### PR TITLE
fix: Release requirements fix

### PR DESCRIPTION
**Pull request by ShubhamChaturvedi7**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -343,7 +343,8 @@ commands =
 
 [testenv:test-release]
 basepython = python3
-skip_install = true
+# Set skip_install false because we need to install the cli for sphinx.
```
Comment (RitvikKapila) [inline tox.ini:346]: Nit:
```suggestion
# Set skip_install false because we need to install the aws-encryption-sdk-cli for sphinx.
```

**Review by RitvikKapila — APPROVED** [review]
Left a nit
Otherwise LGTM
Approved

File: tox.ini (hunk 1 above)
Comment (ShubhamChaturvedi7) [reply tox.ini]: Fair enough. My assumption was that it is obvious enough.

**Review by RitvikKapila — APPROVED** [review]
LGTM

**Pull request merged**